src/actions/MenuActions.js

```diff
@@ -72,10 +72,11 @@ export function queryPathsRequest(query, limit, child) {
     );
 }
 
-export function rootRequest(limit) {
+export function rootRequest(limit, depth) {
     return () => axios.get(
         config.API_URL +
-        '/menu/root?depth=10' +
+        '/menu/root?depth=' +
+        (depth ? depth : '10')  +
```

Comment:
Why the default value is set to 10? Shouldnt we fix 0 as a default value? To understand that like, `always fetch all until you pass something` And if we want to fix some value, shouldn't be better option to use defaults for parameters in functions like:
```
export function rootRequest(limit, depth = 0) {
...
'/menu/root?depth=' +
depth
```

Because rootRequest is (for now) used in one special case (that is using default value). So we are going to save passing some parameters below :)